source/gui/settingsDialogs.py

```diff
@@ -3353,6 +3405,11 @@ class TouchInteractionPanel(SettingsPanel):
 	# Translators: This is the label for the touch interaction settings panel.
 	title = _("Touch Interaction")
 	helpId = "TouchInteraction"
+	panelDescription = _(
+		# Translators: This is a label appearing on the touch interaction settings panel.
+		"Configures touch interaction."
+		" If this category is visible then your system supports touch interaction.",
```

Comment:
I'm not sure if this sentence is helpful, might be worth covering what the settings are instead